docs/draft/microfrontends/create-react-microfrontend-widget.adoc

```diff
@@ -158,11 +158,12 @@ Open the Entando App Builder.
 1. Click save
 1. Click `my-widget` folder
 1. Recreate the same folder structure (my-widget/static/js, my-widget/static/css)
-1. Upload files from js and css folders in the corresponding folders in File Browser
+1. Upload files from js and css folders in the corresponding folders in file browser
```

Comment:
@paulhinz "File Browser" is an actual menu item in the app builder. Not sure of the standard on those but i think you've been capitalizing them elsewhere. I'll merge this one but may want to revisit.